Viz EC2 Services to Lambda - Part 2 - Rapid Onset Flooding Probability

TylerSchrag-NOAA opened the pull request:
This PR migrates the ensemble-based Rapid Onset Flooding Probability products from the Viz EC2 to the Viz Max Values lambda function (now called Python Preprocessing). It broadly encapsulates the following changes:

**Changes**
- New rapid_onset_flooding product script in the python_preprocessing lambda function that supports both 12 hour SRF and 5 day GFS MRF configurations.
- Removal of rapid_onset_flooding pipeline and product files in the source-aws_loosa library

**Deployment Considerations:**
- Not sure if we should include this in the 2.1.4 release or not? I'm good to test/fix quickly and thoroughly next week if you want to include it. Otherwise, fine to go in the next one (wait to merge for now)
- We will need to include a ingest schema db dump when deploying to UAT.

TylerSchrag-NOAA committed:
adding srf and mrf rapid onset flooding probability to viz_python_preprocessing lambda function

TylerSchrag-NOAA committed:
srf and mrf rapid onset flooding probability product sql

TylerSchrag-NOAA committed:
rofp hucs summary SQL + minor tweak to product SQL

TylerSchrag-NOAA committed:
Tweaks to high water probability summary sql

TylerSchrag-NOAA committed:
Bug Fix: HUC table name fix in MRF High Water Probability Mapx

TylerSchrag-NOAA committed:
Viz Initialize Pipeline: Rapid Onset Flooding Probability product config ymls.

TylerSchrag-NOAA committed:
Misc. Fixes

TylerSchrag-NOAA committed:
Data type fix in ROFP mapx files

TylerSchrag-NOAA committed:
Removal of EC2 Rapid Onset Flooding Probability pipelines

CoreyKrewson-NOAA commented on Core/LAMBDA/viz_functions/viz_db_postprocess_sql/summaries/mrf_gfs_5day_max_high_water_probability/hucs.sql:
as in the product is creating a "mrf_gfs_5day_high_water_probability_hucs" table but the mapx is looking for a "mrf_gfs_5day_max_high_water_probability_hucs"

TylerSchrag-NOAA replied:
You're right, I missed/botched this one somehow. Thank you!

CoreyKrewson-NOAA committed:
Update pipelines_config.yml

removed HWP services from the pipelines_config.yml so that the EC2 doesn't create windows services for them to run

CoreyKrewson-NOAA commented:
@TylerSchrag-NOAA I also updated the pipeline_config file which is used by the EC2 to create the windows services

TylerSchrag-NOAA commented:
> @TylerSchrag-NOAA I also updated the pipeline_config file which is used by the EC2 to create the windows services

Thank you! I totally missed that.

shawncrawley reviewed (dismissed):
This looks good to me!

TylerSchrag-NOAA committed:
Addition of new dask layer to python_preprocessing lambda function.

TylerSchrag-NOAA commented:
> This looks good to me!

Thanks Shawn - Sorry to make you approve again, but I just realized I forgot to add the new dask lambda layer.

shawncrawley reviewed (dismissed):
Ah yes, the dask layer. Now it looks better.

TylerSchrag-NOAA committed:
Shortening HWP and ROFP table name to support past event pipelines.

TylerSchrag-NOAA committed:
Fixing rofp csv format - got confused on this and changed this unnecessarily.

TylerSchrag-NOAA committed:
Shortening hwp and rofp table names for srf as well, to enable past event functionality

TylerSchrag-NOAA committed:
Another past event functionality fix

TylerSchrag-NOAA committed:
another hwp hucs fix

TylerSchrag-NOAA committed:
Adding organize_input_files to lambda shared funcs

TylerSchrag-NOAA committed:
Merge branch 'ti' into viz_ec2_services_to_lambda_part2

shawncrawley reviewed (approved):
This looks good to me.
